src/pages/app-development/starter-kit/hooks.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Webhooks example
+description: Learn how to use webhooks with Adobe Commerce Extensibility Starter Kit.
+keywords:
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
+---
+
+# Webhooks example
+
+The following runtime action demonstrates how to expose an entry point to perform a realtime check of the available stock for Commerce cart items. You must implement the stock-checking logic on an external service.
+
+This runtime response uses the [Adobe Commerce webhook module](https://developer.adobe.com/commerce/extensibility/webhooks/) to allow intercepting flows in Adobe Commerce.
```

Comment:
This document already covers responses at the end, maybe this statement could be rephrased to
>  This runtime **action uses the approach described at** [Adobe Commerce webhook module](https://developer.adobe.com/commerce/extensibility/webhooks/) to allow intercepting flows in Adobe Commerce.